src/main/resources/config/proxy.yml

```diff
@@ -8,7 +8,7 @@ http2Enabled: ${proxy.http2Enabled:true}
 httpsEnabled: ${proxy.httpsEnabled:true}
 
 # Target URIs
-hosts: http://localhost:${proxy.containerPort:}
+hosts: ${proxy.host:http://localhost:8080}
```

Comment:
I think here we should use the list instead of a string. In the http-sidecar, we only have one instance of the backend in the pod. However, when using the proxy alone, there would be several backend instances in most of the cases. What do you think? Thanks.

Reply:
Sorry. Thanks for the reminder. I totally forget about the format. It was a list of hosts joined with commas. Yes. adding a comment and updating the readme help. Thanks.